# ускорение по оси X
float32 acc_x

# ускорение по оси Y
float32 acc_y

# ускорение по оси Z
float32 acc_z